src/Providers/TemplatedEmailProvider/NotifyTemplatedEmailProvider.cs

```diff
@@ -1,19 +1,36 @@
-﻿using Notify.Interfaces;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Notify.Interfaces;
 
 namespace form_builder.Providers.TemplatedEmailProvider
 {
     public class NotifyTemplatedEmailProvider : ITemplatedEmailProvider
     {
         private readonly IAsyncNotificationClient _notifyClient;
+        private readonly ILogger<NotifyTemplatedEmailProvider> _logger;
 
-        public NotifyTemplatedEmailProvider(IAsyncNotificationClient notifyClient) => _notifyClient = notifyClient;
+        public NotifyTemplatedEmailProvider(IAsyncNotificationClient notifyClient, ILogger<NotifyTemplatedEmailProvider> logger)
+        {
+            _notifyClient = notifyClient;
+            _logger = logger;
+        }
 
         public string ProviderName { get => "Notify"; }
 
-        public async Task SendEmailAsync(string emailAddress, string templateId, Dictionary<string, dynamic> personalisation) =>
+        public async Task SendEmailAsync(string emailAddress, string templateId,
+            Dictionary<string, dynamic> personalisation)
+        {
+            try
+            {
                 await _notifyClient.SendEmailAsync(emailAddress, templateId, personalisation);
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError($"Notify email provider :: email not sent: {ex.Message}");
```

Comment:
can we include the templateId within the exception, just as i know notify error does not log this currently, will help matching it to form if there is an error.